pink/tasks/relative_frame_task.py

```diff
@@ -86,15 +86,19 @@ def set_position_cost(
         """
         if isinstance(position_cost, float):
             assert position_cost >= 0.0
-        else:  # not isinstance(position_cost, float)
-            assert all(cost >= 0.0 for cost in position_cost)
-        if isinstance(self.cost, np.ndarray):
-            self.cost[0:3] = position_cost
-        else:  # self.cost is not a vector
-            raise TaskDefinitionError(
-                "Frame task cost should be a vector, "
-                f"currently cost={self.cost}"
-            )
+        else:  # Should be a ndarray or seq
+            if not isinstance(position_cost, np.ndarray):  # Must be seq
```

Comment:
Since this is an `else: if`, why not make it an `else if`? (Pro: one level of indentation less.)